src/main/kotlin/com/refinedmods/refinedstorage/extensions/Deprecated.kt

```diff
@@ -0,0 +1,81 @@
+@file:Suppress("unused")
+
+package com.refinedmods.refinedstorage.extensions
+
+import com.refinedmods.refinedstorage.RS
+import com.refinedmods.refinedstorage.apiimpl.storage.ItemStorageType
+import com.thinkslynk.fabric.generated.ItemRegistryGenerated
+import com.thinkslynk.fabric.generated.MyItemGroups
+import net.minecraft.entity.player.PlayerEntity
+import net.minecraft.entity.player.PlayerInventory
+import net.minecraft.inventory.Inventory
+import net.minecraft.item.Item
+import net.minecraft.item.ItemGroup
+import net.minecraft.item.ItemPlacementContext
+import net.minecraft.item.ItemStack
+import net.minecraft.nbt.CompoundTag
+import net.minecraft.nbt.ListTag
+import net.minecraft.util.Hand
+import net.minecraft.util.Identifier
+import net.minecraft.util.hit.BlockHitResult
+import net.minecraft.util.math.Direction
+import net.minecraft.world.World
+import java.util.*
+
+@Deprecated("fabric", replaceWith = ReplaceWith("this.getStackInHand(hand)"))
+fun PlayerEntity.getHeldItem(hand: Hand): ItemStack = getStackInHand(hand)
```

Comment:
This entire file is very smart. Me like.

Reply:
done